### ISSUE TITLE
Missing CHANGELOG.md

### ISSUE THREAD

**Issue by emilk** [ndarilek/tts-rs#10]
When updating a dependency I look for `CHANGELOG.md` as a guide to how to fix upgrade issues (such as "could not find `TTS` in `tts`").

It is also nice to track progress :)

**Comment by ndarilek** [comment]
I agree that a changelog would be useful, but I maintain a whole bunch 
of projects, and this one with its support for multiple platforms is a 
lot bigger than its surface area might lead you to believe. As such, I 
think that maintaining a changelog would increase the maintenance 
burden. If anyone is interested in co-maintaining, then that might be a 
duty I delegate. But I think that having to write up a list of changes 
would be a bit much.


That said, I recognize the burden on downstream users as well. My 
compromise is that I keep the examples updated, and require successful 
builds of all examples before a commit passes CI. I also require that 
any PRs adding new functionality include examples so we can ensure that 
changes don't break anything. If something stops working, the examples 
are a good place to look for why that might be.


I'll leave this issue open in case anyone wants to help with this.
